h2olog

```diff
@@ -868,7 +868,11 @@ if sys.argv[1] == "quic":
 
     # provider h2o:
     u.enable_probe(probe="send_response_header", fn_name="trace_h2o__send_response_header")
-    b = BPF(text=quic_bpf, usdt_contexts=[u])
+
+    probes = filter(lambda probe: probe.provider == "h2o" or probe.provider == "quicly", u.enumerate_probes())
+    max_event_type_len = 1 + max(map(lambda probe: len("%s__%s" % (probe.provider, probe.name)), probes))
+    cflags = ["-DMAX_EVENT_TYPE_LEN=%s" % max_event_type_len]
```

Comment:
Would you mind grouping this in something like `get_event_type_len()`? thanks!

Reply:
Yeah, it should be. done! 56186fea870a2875269f5eb1360d986ee141e4d5